java/code/src/com/suse/manager/reactor/utils/RhelUtils.java

```diff
@@ -280,6 +282,11 @@ public String getRelease() {
             return amazonReleaseFile.map(v -> detectPlainRHEL(v, arch, "AmazonLinux"));
         }
 
+        // next check if Rocky Linux
+        if (rockyReleaseFile.filter(StringUtils::isNotBlank).isPresent()) {
+            return rockyReleaseFile.map(v -> detectPlainRHEL(v, arch, "RockyLinux"));
```

Comment:
Following the code path, the `RockyLinux` value here has no effect at all in the normal case, (it's overridden by the name in the release file) but for consistency, it could be changed to `Rocky`.

The values in the SUSE product for product matching lookup shall be:
Name: `Rocky`
Version: `8` or null
Release: `Green Obsidian` or null
Arch: `x86_64` or null

Reply:
Does it mean we have it wrong for `AmazonLinux` as well?

Reply:
probably, but like I said, as long as the release file is there, it'll be overridden so it's not a big deal.

Reply:
That parameter means "look up this name in SUSE products if I cannot parse the release file for some reason".